Cython/Compiler/Scanning.py

```diff
@@ -330,6 +330,8 @@ def __init__(self, file, filename, parent_scanner=None,
         self.indentation_char = None
         self.bracket_nesting_level = 0
         self.async_enabled = 0
+        if parent_scanner and getattr(parent_scanner, 'async_enabled', 0):
```

Comment:
Could you remove the `getattr()`, please? It should now work with normal attribute access. If not, then that is something to fix, not to work around.

Reply:
ok, forgot it